mifosng-android/src/main/res/layout/fragment_group_details.xml

```diff
@@ -11,7 +11,8 @@
 
         <LinearLayout style="@style/LinearLayout.Base">
 
-            <LinearLayout style="@style/LinearLayout.Width">
+            <LinearLayout style="@style/LinearLayout.Width"
+                android:paddingTop="8dp">
```

Comment:
Try making the UI overall more attractive if you are making any UI related changes, I think adding only top padding is not justifiable to the PR. If possible please make the UI better

Reply:
I thought of but changing ui might needs some approval so i went with padding only. Should i change some of items?

Reply:
Ok i am working on it.